File: zgl-dao/src/main/java/com/zgl/service/ThreeService.java
package com.zgl.service;
import java.util.List;
import com.zgl.pojo.TbThree;

import com.zgl.pojo.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface ThreeService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbThree> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbThree three);
	
	
	/**
	 * 修改
	 */
	public void update(TbThree three);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbThree findOne(Integer id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Integer[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbThree three, int pageNum, int pageSize);
	
}

File: zgl-web/src/main/webapp/js/service/threeService.js
//服务层
app.service('threeService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../three/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../three/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../three/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../three/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../three/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../three/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../three/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});
